pkg/typesystem/typesystem.go

```diff
@@ -207,15 +228,26 @@ func New(model *openfgav1.AuthorizationModel) *TypeSystem {
 			WithMaxEvaluationCost(config.MaxConditionEvaluationCost()).
 			WithInterruptCheckFrequency(config.DefaultInterruptCheckFrequency)
 	}
+	authorizationModelGraph, err := graph.NewAuthorizationModelGraph(model)
```

Comment:
By default, the direction of this graph is for ListObjects (added docs for this here https://github.com/openfga/language/pull/353)

is this what we want? if not, could we add code to be 100% sure that it's like that?

```go
authorizationModelGraph, err := graph.NewAuthorizationModelGraph(model)
if err != nil {
	return err
}
if (authorizationModelGraph.drawingDirection != drawingDirectionListObjects) {
     authorizationModelGraph = authorizationModelGraph.Reversed()

}

Reply:
Good callout.  I wrote the logic with ListObjects order - we should assert that.

Reply:
See https://github.com/openfga/language/pull/353#discussion_r1769162266, we need the method to expose the drawing direction as this is a private member.

Reply:
Done and merged :)